protected/humhub/components/export/SpreadsheetExport.php

```diff
@@ -136,7 +137,7 @@ public function init()
             ]);
         }
 
-        if ($this->dataProvider instanceof ActiveDataProvider) {
+        if ($this->dataProvider instanceof ActiveDataProvider || $this->dataProvider instanceof ArrayDataProvider) {
```

Comment:
Just `if ($this->dataProvider instanceof BaseDataProvider) {` should be correct here.